pywr/optimisation/__init__.py

```diff
@@ -60,6 +60,7 @@ class BaseOptimisationWrapper(object):
     def __init__(self, pywr_model_json, *args, **kwargs):
         uid = kwargs.pop("uid", None)
         self.pywr_model_klass = kwargs.pop("model_klass", Model)
+        self.pywr_model_options = kwargs.pop("model_options", {})
```

Comment:
On second thought, could we rename this to be `model_kwargs`?

Reply:
Sorted